evalprompt/genSpecFromApi/promptfooconfig.yaml

```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://promptfoo.dev/config-schema.json
+prompts:
+  - file://prompts.js
+
+providers:
+  - id: azure:chat:gpt-4o
+    config:
+      apiHost: "apicevaluationtestai.openai.azure.com"
```

Comment:
P2: Better to read from variable?
We could improve this later

Reply:
sure, the endpoint may not a secrets, will improve this later